runelite-client/src/main/java/net/runelite/client/util/Text.java

```diff
@@ -48,10 +48,26 @@ public static String removeTags(String str)
 	/**
 	 * In addition to removing all tags, replaces nbsp with space, trims string and lowercases it
 	 * @param str The string to standardize
+	 *
 	 * @return The given `str` that is standardized
 	 */
 	public static String standardize(String str)
 	{
-		return Text.removeTags(str).replace('\u00A0', ' ').trim().toLowerCase();
+		return removeTags(str).replace('\u00A0', ' ').trim().toLowerCase();
+	}
+
+	/**
+	 * In addition to removing all tags, replaces all <br> delimited text with spaces and all multiple continuous
+	 * spaces with single space
+	 *
+	 * @param str The string to sanitize
+	 * @return sanitized string
+	 */
+	public static String sanitizeMultilineText(String str)
+	{
+		return removeTags(str
+			.replaceAll("-<br>", "-")
+			.replaceAll("<br>", " ")
+			.replaceAll("[ ]+", " "));
```

Comment:
Is this doing a regular expression match for spaces? could you use `\s+`?

Reply:
it is explicitely removing duplicate spaces, nothing else matters in this case